### PR TITLE
Docs-3050 13.0-U2 Release Notes

### PR DESCRIPTION
**Pull request by micjohnson777**
13.0-U2 Release Notes initial commit for downloads as of 8/18 and the RealTek issue added to known issues. 

This is the first commit. Updates to follow in a few days.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Review by tonyriv3 — APPROVED** [review]
LGTM

File: content/CORE/COREReleaseNotes.md [hunk 1]
```diff
@@ -21,13 +22,84 @@ aliases:
 
 | Version | Checkpoint | Scheduled Date |
 |---------|------------|----------------|
+| 13.0-U3 | Release | To be determined |
 | 13.0-U2 | Code-freeze | 17 August 2022 |
 | 13.0-U2 | Internal Testing Sprints | 22 August - 26 August 2022 |
 | 13.0-U2 | Tag | 29 August 2022 |
 | 13.0-U2 | Release | 30 August 2022 |
```
Comment (Mrt134) [inline content/CORE/COREReleaseNotes.md:29]: these can be removed
```suggestion
```

File: content/CORE/COREReleaseNotes.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ aliases:
   - /releasenotes/core/13.0beta1/
   - /releasenotes/core/13.0rc1/
   - /releasenotes/core/
+  - /core/corereleasenotes/
```
Comment (Mrt134) [inline content/CORE/COREReleaseNotes.md:8]: This isn't needed
```suggestion
```

**Review by Mrt134 — COMMENTED** [review]
Couple of cleanup things, but otherwise looks good.

**Review by micjohnson777 — COMMENTED** [review]
Suggested changes committed. This is ready when approved.

**Review by Mrt134 — APPROVED** [review]

**Review by ericbsd — APPROVED** [review]

**Pull request merged**